components/tiles/types.ts

```diff
@@ -0,0 +1,7 @@
+import { Network } from "../button/types";
+
+interface ILBTiles {
+  network: Network;
+}
+
+export type { ILBTiles };
```

Comment:
no need to export

Reply:
When you import something from another file, in this case, I've imported `Network` from another file, your current file becomes a module, and you must explicitly export interfaces or types to use them elsewhere. Without imports, everything stays within a simpler local scope, allowing you to use interfaces directly without needing to export them.